main.py

```diff
@@ -10,14 +10,18 @@
 from data.postgres import PostgresDataBase
 
 @click.command()
-@click.option('--data_file', help = 'Path to datafile.')
+@click.option('--data_source', help = 'Data source for persisting data. Can be either path to csv or connection string to postgres database.')
 @click.option('--break_time', default = 5, help = 'Time of a break between prices.')
-def main(data_file, break_time):
-    # Clear all data from 'bitcoin_price.csv'
-    csv_file = CsvFile(data_file)
-    csv_file.remove_data_file()
-
-    postgres = PostgresDataBase()
+def main(data_source, break_time):
+    if data_source.startswith("postgres://"):
+        # Clear all data from 'postgres data base'
+        postgres = PostgresDataBase()
```

Comment:
in order to benefit fully from abstract class we can write something like:
```
    if data_source_startswith("postgres://"):
	datamanager = PostgresDataBase()
    else:
        datamanager = CsvFile(data_source)
        
        
        datamanager.clean()
        
        ...
        datamanager.insert(...)
```

as a benefit you don't need second if in line 60.